lib/util.ml

```diff
@@ -30,6 +30,14 @@ module Encoding = struct
   let unpack reader payload =
     let buf = prepare_buff payload in
     read_all reader buf
+
+  let str_dump payload =
```

Comment:
I know this function is a basic transformation, is it possible to add a comment after some important lines to explain what is the related step?
Or maybe just an example of `payload -> result`

Reply:
Can you explain a little more what you want me to comment on?

Reply:
Re-thinking about it, a comment is useless. At least a unitary test would be more helpful.

Reply:
Will be done within #19